test/web/10-api.js

```diff
@@ -182,6 +182,7 @@ describe('Profile Manager API', () => {
         kmsModule: KMS_MODULE,
         kmsBaseUrl: KMS_BASE_URL,
         edvBaseUrl: `https://localhost:18443/edvs`,
+        // edvBaseUrl: `${window.location.origin}/edvs`,
```

Comment:
@dlongley we discussed making this change throughout here, but test fail if this baseUrl is used.  Perhaps this points to additional issues in our implementation?  I'm going to collect additional information about the failure case now.

Reply:
When L185 is used we hit an error here because `localhost:9xxx` (karma) does not correspond to config.server.host in bedrock-edv-storage.

https://github.com/digitalbazaar/bedrock-edv-storage/blob/master/lib/helpers.js#L61-L79

Reply:
Hmm, will think about this more. It may be that we want to set `config.server.host = 'localhost:9876'` to handle this case. It's different from the KMS one. Can you see if we did that if all the tests would pass or if we'd end up in some other broken state?

Reply:
Yes, that change works, I made a PR: https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/68